### PR TITLE
protected method in the controller

### PR DESCRIPTION
**Pull request by zagovorichev**

### CODE REVIEW

File: actions/class.RestTests.php [hunk 1]
```diff
@@ -22,6 +22,10 @@
  */
 class taoTests_actions_RestTests extends tao_actions_CommonRestModule {
 
+    /**
+     * taoTests_actions_RestTests constructor.
+     * @\oat\tao\model\routing\AnnotationReader\security("hide")
```
Comment (martijn-tao) [inline actions/class.RestTests.php]: Please import the class to prevent FQN usage

**Comment by zagovorichev** [comment]
@martijn-tao Update

**Comment by zagovorichev** [comment]
@martijn-tao Update

File: actions/class.Tests.php [hunk 2]
```diff
@@ -34,7 +36,11 @@
  *
  */
 class taoTests_actions_Tests extends tao_actions_SaSModule {
-    use EventManagerAwareTrait;
+
+    protected function getEventManager()
```
Comment (martijn-tao) [inline actions/class.Tests.php]: Please add a docblock

**Pull request merged**